noir-projects/aztec-nr/aztec/src/encrypted_logs/payload.nr

```diff
@@ -191,4 +189,73 @@ mod test {
         }
         assert_eq(encrypted_note_log_from_typescript.len(), log.len());
     }
+
+    #[test]
+    fn test_incoming_body_ciphertext_matches_typescript() {
```

Comment:
This is a simplified version of a test originally in incoming_body.nr. Simplified because now it has plaintext on input instead of a note.